charts/dependabot-gitlab/values.yaml

```diff
@@ -95,9 +95,9 @@ worker:
     # -- Startup probe failure threshold
     failureThreshold: 12
     # -- Startup probe period
-    periodSeconds: 5
+    periodSeconds: 10
     # -- Startup probe timeout
-    timeoutSeconds: 3
+    timeoutSeconds: 10
```

Comment:
Same as in web service. Instead of timeout increase, I think failureThreshold should have a larger value to ping more times rather than simply waiting longer because service is not up yet and is not responding at all.

Reply:
every day at 7 am all dependabot updates are triggered, then the worker pod is running under maximum load, which requires a larger timeout. With the default settings the pod is in a restart loop until all scheduled work is done.

Reply:
Well, it's a specific use case to your configuration where all updates are triggered at the same time and environment doesn't have enough resources to handle that. It doesn't exactly validate a default value change. That's why there is an option to configure these values according to ones needs and setup.

Like I already mentioned, I do not have such problems at all.